4-WebApp-your-API/4-2-B2C/Client/appsettings.json

```diff
@@ -1,17 +1,23 @@
 ﻿{
-  "AzureAdB2C": {
-    "Instance": "https://fabrikamb2c.b2clogin.com",
-    "ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
-    "Domain": "fabrikamb2c.onmicrosoft.com",
-    "SignedOutCallbackPath": "/signout/B2C_1_susi",
-    "SignUpSignInPolicyId": "b2c_1_susi",
-    "EditProfilePolicyId": "b2c_1_edit_profile" // Optional profile editing policy
-    //"CallbackPath": "/signin/B2C_1_sign_up_in"  // defaults to /signin-oidc
-  },
-  "Logging": {
-    "LogLevel": {
-      "Default": "Warning"
-    }
-  },
-  "AllowedHosts": "*"
+    "AzureAdB2C": {
+        "Instance": "https://<your-b2c-name>.b2clogin.com",
```

Comment:
>your-b2c-name [](start = 30, length = 13)

could we call this "b2c-tenant-name"? we use "tenantName" in our documentation...and thoughts on how to handle a custom policy? Maybe have a comment or something on that.

Reply:
@jennyf19 I changed it to "b2c-tenant-name", but I didn't understand your second comment about custom policy. Could you explain more about the custom policy concerns?

Reply:
@TiagoBrenck sure...i actually meant custom domains, but [custom policies](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-get-started-custom?tabs=applications) are how the developers customize the entire auth experience. it's pretty complicated but very interesting.
custom domains are specific certs given to customers which are unique authorities, like https://cats.com or https://cats.com/{tenantId-guid} so the microsoft branding is completely hidden from the customer. so this customer will not have `*.b2clogin.com`...and we could also have sovereign clouds, like `*.b2clogin.de`...do we need examples for those as well?